src/languages/coffee.coffee

```diff
@@ -1032,11 +1032,11 @@ fixCoffeeScriptError = (lines, e) ->
 fixQuotedString = (lines) ->
   line = lines[0]
   quotechar = if /^"|"$/.test(line) then '"' else "'"
-  if line.getCharAt(0) is quotechar
+  if line.charAt(0) is quotechar
     line = line.substr(1)
-  if line.getCharAt(line.length - 1) is quotechar
-    line = line.substr(0, line.length -1)
-  return quoteAndCEscape fixQuotedLine line
```

Comment:
Hm, seems like it was previously dead code!

This should read: quoteAndCEscape looseCUnescape(line), quotechar